### PR TITLE
doc/ceph-volume: improve front matter

### PR DESCRIPTION
**Pull request by zdover23**
Simplify and improve the readability of the front matter of doc/ceph-volume/lvm/activate.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/ceph-volume/lvm/activate.rst [hunk 1]
```diff
@@ -3,18 +3,19 @@
...
 
-Once :ref:`ceph-volume-lvm-prepare` is completed, and all the various steps
-that entails are done, the volume is ready to get "activated".
+After :ref:`ceph-volume-lvm-prepare` has completed its run, the volume can be
+activated. 
 
-This activation process enables a systemd unit that persists the OSD ID and its
-UUID (also called ``fsid`` in Ceph CLI tools), so that at boot time it can
-understand what OSD is enabled and needs to be mounted.
+The activation of the volume involves enabling a systemd unit that persists the
```
Comment (anthonyeleven) [inline doc/ceph-volume/lvm/activate.rst:9]: Activating the volume involves

Also I'm a markup slut so I'd backtick systemd, OSD ID, and UUID

**Review by anthonyeleven — APPROVED** [review]

File: doc/ceph-volume/lvm/activate.rst (hunk 1 above)
Comment (zdover23) [reply doc/ceph-volume/lvm/activate.rst:9]: Accepted and... uh, indulged.

Comment (anthonyeleven) [reply doc/ceph-volume/lvm/activate.rst]: That style guide we keep talking about ;)

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/54235 - Reef backport
https://github.com/ceph/ceph/pull/54236 - Quincy backport

File: doc/ceph-volume/lvm/activate.rst (hunk 1 above)
Comment (zdover23) [reply doc/ceph-volume/lvm/activate.rst]: I'll announce at the 2024 Cephalocon that we'll have a style guide by 2025.

**Pull request merged**